cooking-user/sj2324-5ehif-cooking-user.Webapi/Properties/launchSettings.json

```diff
@@ -26,15 +27,19 @@
       "launchUrl": "swagger",
       "applicationUrl": "https://localhost:7231;http://localhost:5069",
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "aspnetcore_enviroment": "Development"
```

Comment:
Probably there is a reason for it, but by default  ASP.NET Core looks for the ASPNETCORE_ENVIRONMENT and apperantly it is case sensitive